examples/installation/single_account_deployment/variables.tf

```diff
@@ -24,6 +24,18 @@ variable "sonar_version" {
   }
 }
 
+variable "additional_tags" {
+  type        = list(string)
+  default     = []
+  description = "Additional tags to add to the DSFKit resources. Please put tags in the following format - Key: Name. For example - [\"Key1=Name1\", \"Key2=Name2\"]"
+  validation {
+    condition = alltrue([
+    for tag_pair in var.additional_tags : can(regex("^([a-zA-Z0-9+\\-_.:/@]+)=([a-zA-Z0-9+\\-_.:/]+)$", tag_pair))
+    ])
+    error_message = "Invalid tag format. All values must be in the format of 'key=value', where 'key' is a valid AWS tag name and 'value' is a valid AWS tag value."
```

Comment:
Nice, refraining from mentioning AWS validation rules since they can change them. But I think we should mention that = is invalid for us although it's valid for AWS. For example, add: (With the the exception of the '=' character which is invalid).